src/Web/Slack/Common.hs

```diff
@@ -19,6 +20,10 @@ module Web.Slack.Common
     , mkHistoryReq
     , HistoryRsp(..)
     , Message(..)
+    , Color
+    , unColor
+    , UserId
+    , unUserId
```

Comment:
Why not `Color(..)` and `UserId(..)` (or `Color(unColor)` and `UserId(unUserId)`)?